gamemodes/terrortown/gamemode/client/cl_help/populate_equipment.lua

```diff
@@ -0,0 +1,33 @@
+local materialIcon = Material("vgui/ttt/vskin/helpscreen/equipment")
+
+---
+-- Function to populate main menu
+-- @param table helpData The main menu data object
+-- @param string id The unique id of this menu element
+-- @internal
+-- @realm client
+HELPSCRN.populate["ttt2_equipment"] = function(helpData, id)
+	local equipmentData = helpData:RegisterSubMenu(id)
+
+	equipmentData:SetTitle("menu_equipment_title")
+	equipmentData:SetDescription("menu_equipment_description")
+	equipmentData:SetIcon(materialIcon)
+
+	equipmentData:AdminOnly(true)
+end
+
+---
+-- Function to populate sub menu
+-- @param table helpData The sub menu data object
+-- @param string id The unique id of this menu element
+-- @internal
+-- @realm client
+HELPSCRN.subPopulate["ttt2_equipment"] = function(helpData, id)
+	-- IDEA: submenu is populated with equipment, each thing gets their
+	-- own menu page
+
+	--local hudData = helpData:PopulateSubMenu(id .. "_hud")
+
+	--hudData:SetTitle("submenu_administration_hud_title")
+	--hudData:PopulatePanel(PopulateHUDPanel)
```

Comment:
Please remove commented code

Reply:
I can remove it, but I left in in there for you to have a baseline

Reply:
Since other menues are already implemented it is not needed, imo.